SwiftIO/SocketPacket.swift

```diff
@@ -217,19 +216,19 @@ class SocketPacket {
             }
         } else if let dict = data as? NSDictionary {
             var newDict = NSMutableDictionary(dictionary: dict)
-            
+
             for (key, value) in dict {
-                newDict[key as NSCopying] = _fillInPlaceholders(value)
+                newDict[key as! NSCopying] = _fillInPlaceholders(value)
```

Comment:
Using Swift 1.2 the compiler says: 'NSCopying' is ambiguous for type lookup in this context.